internal/core/data/event.go

```diff
@@ -208,10 +212,25 @@ func putEventOnQueue(e contract.Event, ctx context.Context) {
 	evt := models.Event{}
 	evt.Event = e
 	evt.CorrelationId = correlation.FromContext(ctx)
-	err := ep.SendEventMessage(evt)
+	payload, err := json.Marshal(evt)
 	if err != nil {
-		LoggingClient.Error("Unable to send message for event: " + e.String())
+		LoggingClient.Error("Unable to marshal event to json: " + err.Error() + ", Event: " + evt.String())
```

Comment:
Please use key and value style for logging and not string concatenation.

Reply:
will do

Reply:
@brandonforster i can't find any examples in this file. could you point me to an example of the preferred logging style? 
does this work?... "LoggingClient.Error(fmt.Sprintf("Unable to marshal event to json: %s, Event: %s", err.Error(), evt.String()))"

Reply:
@brandonforster Are you referring to the Event that Eno has put into the message? If so, I expressed the opinion via Slack that including the Event in this message was too verbose. If we wanted to log the Event, I would suggest doing it as an Info() level statement which can be logged according to the config setting.

Reply:
i think just logging the error should be fine. will make the change.